tests/ios/xcodeproj/post_build_check.sh

```diff
@@ -1,4 +1,4 @@
-set -eux
+set -e
```

Comment:
why this change?

Reply:
i don't think these 2 scripts should output all the execution details, especially when the list of file paths to search can be long. Usually this script output nothing unless there is a path not found

Reply:
still would want `u` then, so we error on unset variables?

Reply:
got it!